src/palace/manager/api/controller/playtime_entries.py

```diff
@@ -49,8 +62,37 @@ def track_playtimes(self, collection_id, identifier_type, identifier_idn):
         except ValidationError as ex:
             return INVALID_INPUT.detailed(ex.json())
 
+        min_time_entry = min([x.during_minute for x in data.time_entries])
+        max_time_entry = max([x.during_minute for x in data.time_entries])
+
+        default_loan_period = timedelta(
+            collection.default_loan_period(
+                library=library, medium=EditionConstants.AUDIO_MEDIUM
+            )
+        )
+
+        loan = self._db.execute(
+            select(Loan)
+            .select_from(Loan)
+            .join(LicensePool)
+            .where(
+                LicensePool.identifier == identifier,
+                Loan.patron == flask.request.patron,
+                Loan.start >= min_time_entry,
+                Loan.start + default_loan_period < max_time_entry,
```

Comment:
It doesn't look like we create any loans in our tests. If not, I'm not sure how the `loan` lookup gets tested.

Reply:
I'll fix that.